packages/frontend-shared/src/locales/en-US.json

```diff
@@ -504,26 +504,25 @@
   "migration": {
     "before": "Before",
     "after": "After",
-    "heresWhy": "here's why:",
     "renameAuto": {
-      "title": "We'll automatically rename your specs in this step",
+      "title": "If you change your default spec pattern, here's what you'll get...",
       "changeButton": "change",
       "changedSpecFolder": "We've changed the default spec folder from:",
       "changedSpecExt": "We've changed the default spec file extension from:",
       "changedSpecPatternExplain": "We've made the default spec file extension to {0} in order to avoid conflicts with any existing testing frameworks.",
       "optedOutMessage": "You've opted not to rename your spec file extension",
       "folderRenameMessage": "You've opted not to rename your spec file extension, we're going to rename the folder",
       "modals": {
-        "title": "Change the existing spec file extension",
+        "title": "Migrating to a new spec file pattern",
         "step1": {
-          "warning": "We recommend using the default extension to avoid inconsistencies, framework conflicts, and confusion with your team.",
-          "line1": "Cypress now supports the ability to create new spec files from within the UI for both E2E and component specs.",
+          "warning": "We strongly recommend having your spec files end in *.cy.* to avoid framework conflicts.",
+          "line1": "You can create new specs without leaving your browser.",
```

Comment:
Should we be adding the periods at the end of these bullets? They don't seem consistent. I didn't notice them on the jira spec